templates/search/blocks/featured-search__interpretive-card.html

```diff
@@ -1,22 +1,33 @@
-<div>
-    <a href="#" aria-hidden="true" tabindex="-1" class="featured-search__interpretive-card-link">
-        <div>
-            <picture class="featured-search__interpretive-card-image">
-                <source media="(max-width: 768px)" srcset="https://via.placeholder.com/508x304">
-                <source media="(max-width: 991px)" srcset="https://via.placeholder.com/268x160">
-                <source media="(max-width: 1199px)" srcset="https://via.placeholder.com/348x208">
-                <source media="(min-width: 1200px)" srcset="https://via.placeholder.com/348x208">
-                <img src="https://via.placeholder.com/348x208" alt="" class="featured-search__interpretive-card-image-fallback"> 
-            </picture>
-        </div>
-    </a>
+{% load search_tags %}
+{% load humanize %}
+
+{% if result.hits.hits.0 %}
+    {% with record=result.hits.hits.0 %}
```

Comment:
Template logic definitely shouldn't be having to contend with elasticsearch-like structures... the API client should be containing that mess to stop it leaking out into the rest of the codebase. We can't do anything about that now though - later maybe :)